code/modules/food_and_drink/ingredients.dm

```diff
@@ -19,7 +19,7 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/ingredient)
 
 	heal(var/mob/living/M)
 		..()
-		if (!(istype(M, /mob/living/critter/plant/maneater)) && prob(33))
+		if (!(istype(M, /mob/living/critter/plant/maneater)) && !(src.name == "bacon") && prob(33))
```

Comment:
Name checks are incredibly brittle, for example labellers can modify the name directly. If you want cooked bacon to not inherit raw meat code then it probably shouldn't inherit from raw meat!

Reply:
for what it's worth i think editing the superclass is the correct approach. making meat ingredients not explicitly poisonous (like giving meat ingredients a 'poisonous' var) makes sense given that both cooked and uncooked meats can be valid in the same cooking recipes. so you can just use a 'snacks/ingredient/bacon' check.